### PR TITLE
Igal/mgmt 1363

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: src/installer/installer.go [hunk 1]
```diff
@@ -355,3 +366,56 @@ func (i *installer) cleanupInstallDevice() error {
...
+		case <-ticker.C:
+			i.log.Infof("searching for hosts that had pulled ignition already")
+			if inventoryHostsMapWithIp == nil {
+				inventoryHostsMapWithIp, err = i.inventoryClient.GetEnabledIdsIps()
+				if err != nil {
+					i.log.Warnf("Failed to get hosts info from inventory, err %s", err)
+					continue
+				}
+			}
+			i.verifyHostCanMoveToConfigurationStatus(&inventoryHostsMapWithIp)
```
Comment (romfreiman) [inline src/installer/installer.go:418]: meaning u gonna read this file every time? no -f option?
btw, what will happen for workers & bootstrap?

Comment (tsorya) [reply src/installer/installer.go:418]: First of all file is really small. Verified it. So didn't really tried to solved this issue.

Regarding workers and bootstrap.
All nodes that will finish reboot and pull ignition from mcs that run on bootstrap will be moved to configuring (if name is ok) state by assisted-installer.
For others we have another ticket, we will need to implement same stuff in assisted-controller, that will read logs of mcs pod in openshift cluster

File: src/ops/ops.go [hunk 2]
```diff
@@ -259,3 +260,24 @@ func (o *ops) RemovePV(pvName string) error {
 	}
 	return err
 }
+
+func (o *ops) GetMCSLogs() (string, error) {
```
Comment (yevgeny-shnaidman) [inline src/ops/ops.go:264]: i think you should add a comment  that explains what we talked about: why we look only at first file, why it is enough etc'

File: src/installer/installer.go [hunk 3]
```diff
@@ -355,3 +366,56 @@ func (i *installer) cleanupInstallDevice() error {
 
 	return i.ops.RemovePV(i.Device)
 }
+
+func (i *installer) verifyHostCanMoveToConfigurationStatus(inventoryHostsMapWithIp *map[string][]string) {
```
Comment (yevgeny-shnaidman) [inline src/installer/installer.go:370]: why do you need to pass the map by pointer, and then de-reference it? map is a reference anyway , and deleting while iterating is also safe

**Comment by ErezAlster** [comment]
#1

File: src/ops/ops.go (hunk 2 above)
Comment (tsorya) [reply src/ops/ops.go:264]: ok

**Pull request merged**